templates/primary-secondary/primary-secondary.js

```diff
@@ -386,12 +389,13 @@ class MobileMouseResizer extends Resizer {
 		const y = e.clientY - this.contentRect.top;
 		const desiredSecondaryHeight = this.contentRect.height - y + this._offset;
 		if (
-			(this._wasCollapsed && desiredSecondaryHeight < collapsedCollapseThreshold)
-			|| (!this._wasCollapsed && desiredSecondaryHeight < expandedCollapseThreshold)
+			this._isDragging
+			&& ((this._wasCollapsed && desiredSecondaryHeight < collapsedCollapseThreshold)
+			|| (!this._wasCollapsed && desiredSecondaryHeight < expandedCollapseThreshold))
 		) {
-			if (desiredSecondaryHeight > 0) this.dispatchResize(0, true);
+			if (desiredSecondaryHeight !== 0) this.dispatchResize(0, true);
 		}
-		else if (desiredSecondaryHeight < this.contentBounds.minHeight) {
+		else if (this._isDragging && desiredSecondaryHeight < this.contentBounds.minHeight) {
```

Comment:
There appears to be some rounding down in the actual height and `this.contentBounds.minHeight`. This resulted in `desiredSecondaryHeight < this.contentBounds.minHeight` being true when the secondary panel was at the min height. 

Dispatching a resize turns off the click action to open or close the panel and since this check was true, it was disabling the close action when clicking at minimum width. I added a check to not resize if the mouse was not moved, so the close panel click action works in that case.